plone/app/discussion/conversation.py

```diff
@@ -90,11 +90,19 @@ def enabled(self):
     @property
     def total_comments(self):
         public_comments = [
-            x for x in self._comments.values()
+            x for x in self.values()
             if user_nobody.has_permission('View', x)
         ]
         return len(public_comments)
 
+    def another_total_comments(self):
```

Comment:
The issue is described here: https://github.com/plone/Products.CMFPlone/issues/324

I created another one as a proof of concept, I just wanted to give a head ups to know if this approach was good or not and if so, then provide a better solution.

I'm still puzzled about the ``@property`` being the problem here...